### DISCUSSION TITLE
Magazine Blog Taxonomy

### DISCUSSION THREAD

**Discussion by theunearther** [11ty/eleventy#1499] [Q&A] [answered]
Say I want to make a magazine style blog with the following structure:

/ - a home page listing the current issue
/issues - an archive page listing all issues
/{issue} - lists the issue 
/{issue}/an-article - post page

Let's say that any issue has a number of sections, let's say: Editorial, Music, Sports, etc. So, for example, 'Issue One' would have a list page with articles under each section.

I'd also like to be able to list, say, all 'Music' articles across issues.

I'm interested to know how others might go about implementing this with Eleventy Collections, tags, or globs.

Thanks
Ryan

**Comment by ItaiAxelrad** [comment, marked as answer]
I would use this file architecture
```
src
- index.njk
- issues
  - issues.njk (archive page)
  - issues.json (add default tags & layouts)
  - music
    - music.json (add default tags & layouts)
    - an-article
       - uploads
       - index.md
  - sports
    - sports.json (add default tags & layouts)
    - another-article
       - uploads
       - index.md
```
Which would yield:
 
`/` home page
`/issues` issue archive page
`/issues/{issue}` specific issue
`/issues/{issue}/an-article/index.html` post page

The eleventy starter blog has an example of an archive page FYI

**Reply by binyamin** [reply]
@ecotyped If you're satisfied with this answer, could you mark it as "accepted" please?

**Reply by theunearther** [reply]
Yeah, I reacted instead of marking. Thanks to @ItaiAxelrad, much appreciated.
